### PR TITLE
always show community name with instance for easy copy-pasting to friends

### PR DESCRIPTION
**Pull request by Die4Ever**
## Description

I don't like the inconsistency of community names currently, the UI shouldn't be presenting things differently because of technical reasons that are hard for users to understand. This makes it easier to share communities with your friends!

### Before

image

### After

image

### CODE REVIEW

**Review by SleeplessOne1917 — APPROVED** [review]
LGTM. Will wait for approval from the maintainers before I merge.

File: src/shared/components/community/community-link.tsx
Comment (camporter) [inline src/shared/components/community/community-link.tsx:28]: `_name` isn't really used now.

Comment (Die4Ever) [reply src/shared/components/community/community-link.tsx]: thanks, fixed

File: src/shared/components/community/community-link.tsx
Comment (dessalines) [inline src/shared/components/community/community-link.tsx:36]: This *should* be okay, because it should only use the apubName on community pages, hopefully not in most community links.

**Review by dessalines — APPROVED** [review]

**Pull request merged**
